modin/pandas/test/test_io.py

```diff
@@ -199,6 +200,26 @@ def eval_to_file(modin_obj, pandas_obj, fn, extension, **fn_kwargs):
         teardown_test_files([unique_filename_modin, unique_filename_pandas])
 
 
+@pytest.fixture
+def make_parquet_dir():
+    list_of_path: list[str] = []
+
+    def _make_parquet_dir(
+        path: str, dfs_by_filename: Dict[str, pandas.DataFrame], row_group_size: int
+    ):
+        if os.path.exists(path):
+            shutil.rmtree(path)
+        else:
+            os.makedirs(path)
```

Comment:
```suggestion
        os.makedirs(path)
```
otherwise if `path` exists you'd wipe it but won't make an empty directory to save to